books.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def rent_books(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] This method is for adding `rentals` and remembering `the book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association and also we need a `person` who want to rent and `date`
```ruby
def rent_books(person, date)
    Rental.new(date, self, person)
  end
```